src/Presentation/ListFacetHtmlBuilder.php

```diff
@@ -119,6 +127,8 @@ private function buildCheckboxesViewModel( FacetConfig $config, PropertyConstrai
 			$checkboxes[] = $this->buildCheckboxViewModel( $config, $valueCount, $selectedValues, $state->propertyId, $i );
 		}
 
+		usort( $checkboxes, fn( $a, $b ) => $b['checked'] <=> $a['checked'] );
```

Comment:
Does this keep the current order of the checked items?

Reply:
Yes it does